api-catalog-ui/frontend/src/reducers/wizard-reducer.jsx

```diff
@@ -21,6 +21,7 @@ import {
     VALIDATE_INPUT,
     WIZARD_VISIBILITY_TOGGLE,
     OVERRIDE_DEF,
+    UPDATE_UPLOADED_YAML_TITLE,
```

Comment:
this new reducer need to be tested too, in the `wizard-reducer.test.jsx`, as reported by the coverage analysis